align.go

```diff
@@ -160,25 +201,25 @@ func (a *Align) columnLength() []string {
 			}
 		}
 
-		lines = append(lines, line)
+		a.lines = append(a.lines, line)
 	}
-
-	return lines
 }
 
+const padchar byte = ' '
+
 // export will pad each field in lines based on the Align's column counts.
-func (a *Align) export(lines []string) {
+func (a *Align) export() {
 	if a.padOpts.Pad < 0 {
 		a.padOpts.Pad = 0
 	}
 
-	p := make([]byte, 0, a.padOpts.Pad)
+	surroundingPad := make([]byte, 0, a.padOpts.Pad)
 
 	for i := 0; i < a.padOpts.Pad; i++ {
```

Comment:
I'd be curious what the performance of this is versus `strings.Map`.

Reply:
For this particular loop?  In what way do you mean?  All this loop is doing is creating the configured pad specified by an option.  It also only happens once.  I might not be understanding though.